lang/gcc-devel/Portfile

```diff
@@ -173,7 +156,7 @@ conflicts_build-append libunwind-headers
 # Note that we really don't want to include libgcc_ext.10.[45].dylib here, but install_name_tool
 # doesn't know how to change the id of stubs, and it's easier than recreating them for each
 # gcc port.
-set dylibs {libgcc_ext.10.4.dylib libgcc_ext.10.5.dylib libgcc_s.1.dylib libgfortran.5.dylib libquadmath.0.dylib libstdc++.6.dylib libobjc-gnu.4.dylib libgomp.1.dylib libitm.1.dylib libssp.0.dylib libasan.6.dylib libubsan.1.dylib libatomic.1.dylib}
+set dylibs {libgcc_ext.10.4.dylib libgcc_ext.10.5.dylib libgcc_s.1.dylib libgcc_s.2.dylib libgfortran.5.dylib libquadmath.0.dylib libstdc++.6.dylib libobjc-gnu.4.dylib libgomp.1.dylib libitm.1.dylib libssp.0.dylib libasan.6.dylib libubsan.1.dylib libatomic.1.dylib}
```

Comment:
Well -- now here we have problem that we need to actually sort out. This is very very strange. Will gcc from here on have a new library we need to carry along? I will check with the darwin lead here about this, but I wasn't expecting a new library.

Reply:
It looks like this library was added to the arm64 branch only, three days ago <https://github.com/iains/gcc-darwin-arm64/commit/c29742146282119c5f759d5d544b6bffaaec4362#diff-b39028e53782355458efc72ceb1bd540350adb73d0db52e04372045adbd50d39> so -- yes, we'll need it for Iain's arm64 branch, but it does not exist on the gcc mainline.

Reply:
If the library list is different for the different platforms then the above needs to add this conditionally based on the system arch.

Reply:
Actually, reminding myself a bit of how that list is used, its fine to have lib rabies in it that are not produced as the logic checks for their existence and skips those missing. So its OK to just list anything that could be produced...